erpnext/accounts/doctype/account/chart_of_accounts/verified/scrape_co_chart_of_accounts.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+"""
+This script scrapes the colombian chart of accounts in two variants from https://puc.com.co
```

Comment:
What is the purpose of this, does this even work?

Reply:
@deepeshgarg007 should we keep this file in here or rather push it into a gist or something?

Reply:
Separating it out would be preferred

Reply:
@deepeshgarg007 : we're good I removed that file and placed a link to the gist into the commit message instead.